tc/examples/blockdiagperm.cc

```diff
@@ -116,12 +142,19 @@ def blockdiagperm2dfissioned_2(float(B, N) I, int32(N) Idx) -> (O) {
     tc::aten::uncheckedRun(*pExecutor, {I, W}, outputs);
     tc::aten::uncheckedRun(*pExecutor2, {O, Idx}, outputs2);
   }
+
+  // 5. Explicitly store the best found options in cache so we can reinforce
+  // later if required
+  tc::autotune::storeTopKToCache(
```

Comment:
I'd rather not call "store" if the path is empty explicitly, then have an implicit behavior that does not store (and logs) given an empty path